src/Blazor.EventGridViewer.Core/Models/CloudEvent.cs

```diff
@@ -21,7 +22,7 @@ public class CloudEvent
         public string Id { get; set; }
 
         [JsonProperty("time")]
-        public string Time { get; set; }
+        public DateTimeOffset Time { get; set; }
```

Comment:
Could we just use the CloudEvent type from Azure.Core - https://docs.microsoft.com/en-us/dotnet/api/azure.messaging.cloudevent?view=azure-dotnet